library/erlang

```diff
@@ -34,3 +34,7 @@ Directory: 17
 Tags: 17.5.6.9-slim, 17.5.6-slim, 17.5-slim, 17-slim
 GitCommit: ea32d5f6f1735f9f55bee04b112166da96eb9c73
 Directory: 17/slim
+
+Tags: 17.5.6.9-onbuild, 17.5.6-onbuild, 17.5-onbuild, 17-onbuild
+GitCommit: dbcb2dd16843db0663d10278f2a13cc70a840494
+Directory: 17/onbuild
```

Comment:
I've sent https://github.com/c0b/docker-erlang-otp/pull/20 upstream to take care of this change.